README.md

```diff
@@ -89,17 +103,24 @@ name, type and location.
 import com.metriport.api.resources.medical.organization.types.BaseOrganization;
 
 // Doesn't compile
-BaseOrganization org = BaseOrganization.builder()
-  .name(...)
-  .type(...)
-  .build(); 
+BaseOrganization org = Address.builder()
```

Comment:
`Address address =` ? You'll also want to edit the text above that references BaseOrganization